modules/040-node-manager/docs/internal/static-provider/images/static-instance-state.puml

```diff
@@ -0,0 +1,43 @@
+@startuml
+title Figure 1: State diagram with a StaticInstance
+
+(*) --> [  StaticInstance Create  ] "Pending"
+
+note right
+- StaticInstance controller watches for new StaticInstances,
+  and sets //StaticInstance.status.phase// = "Pending".
+end note
+
+"Pending" --> "Bootstrapping"
+
+note right
+- StaticMachine controller watches for new StaticMachines and
+  starts bootstrapping StaticInstance which is matched by the StaticMachine.spec.labelSelector.
+- StaticMachine controller generates provider ID and
+  sets //StaticMachine.spec.providerID// = "static://{{ random uuid }}".
+- StaticMachine controller sets //StaticInstance.status.phase// = "Bootstrapping".
+- StaticMachine controller connects to the StaticInstance host via SSH and runs the bootstrap script.
+end note
+
+"Bootstrapping" --> "Running"
+
+note right
+- After bootstrapping is completed successfully the StaticMachine controller
```

Comment:
It’s not clear from the sentence how we understand that bootstrap was successful